src/main/groovy/wooga/gradle/release/utils/ReleaseNotesGenerator.groovy

```diff
@@ -171,4 +169,84 @@ class ReleaseNotesGenerator {
             return false
         }
     }
+
+    private List<PacketDependency> fetchTemplateDependencies(GHRepository hub, String packageId, ReleaseVersion version) {
+
+        String[] paths = ["$packageId/paket.template", "paket.template"]
```

Comment:
I would use `List<String>` instead. At least the rest of the project uses the `List` interfaces.